index.js

```diff
@@ -26,6 +26,15 @@ module.exports = {
         s3Client: function(context) {
           return context.s3Client; // if you want to provide your own S3 client to be used instead of one from aws-sdk
         },
+        didDeployMessage: function(context){
+          var revisionKey = context.revisionData && context.revisionData.revisionKey;
+          var activatedRevisionKey = context.revisionData && context.revisionData.activatedRevisionKey;
```

Comment:
Where is `revisionData.activatedRevisionKey` set? In the redis case this is set in the activate hook https://github.com/ember-cli-deploy/ember-cli-deploy-redis/blob/aaea5fb1ba806335e3dfbd56f383ae7728f84549/index.js#L89 which would be easy to do in this plugin as well but seems like to be missing here